iOS internship/ForecastModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ForecastModel.swift
+//  iOS internship
+//
+//  Created by pavels.vetlugins on 26/09/2018.
+//  Copyright © 2018 pavels.vetlugins. All rights reserved.
+//
+
+import Foundation
+import Freddy
+
+public class ForecastModel: FetchForecastDelegate{
+    
+    var delegate: GetForecastDelegate?
+
+    let forecastTransport = ForecastTransport()
+    
+    init() {
+        forecastTransport.delegate = self
+    }
+    
+    func fetchForecast(fetchLocation location: String){
+        forecastTransport.fetchForecast(fetchLocation: location)
+    }
+    
+    
+    func forecastFetched(forecast: Forecast) {
+        guard let delegate = self.delegate else {
+            return
+        }
+        delegate.getForecast(forecast: forecast)
```

Comment:
Again, naming, you are not "getting" forecast here, you are setting it, this method should be something like `delegate.update(forecast: forecast)`.